lib/errors.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
```

Comment:
string interpolation IMO is one of the best features in ES6! I get that it may be slightly unreadable for someone unfamiliar with the syntax, especially when used with an arrow function and implicit return, but I'd hope most users would be able see what's happening